### ISSUE TITLE
Unable to Reproduce FuncQA One-Hop Results

### ISSUE THREAD

**Issue by chenhongqiao** [Ber666/ToolkenGPT#12]
Running the inference script with the official checkpoint yields an accuracy of 58.3% on FuncQA one-hop, different from the 73% claimed in the paper. Any thoughts?

[Output](https://pastebin.com/JQA0aGjQ)

**Comment by Leolty** [comment]
Hi there,

Apologies for the confusion. There is indeed a typo regarding the one-hop. We may not want to encourage too frequent function calls, so adjusting the logits-bias can help. I've tested with a bias of ~3, and it should give results > 70%.

Specifically, you may change `--logits_bias 4.0` to `--logits_bias 3.0`.

Thank you for pointing it out and let us know if you encounter any further issues.
